static/app/views/alerts/rules/metric/triggers/actionsPanel/index.tsx

```diff
@@ -436,6 +459,27 @@ class ActionsPanel extends PureComponent<Props> {
                         'inputChannelId'
                       )}
                     />
+                    {hasPriorityFlag &&
+                    availableAction &&
+                    (availableAction.type === 'opsgenie' ||
+                      availableAction.type === 'pagerduty') ? (
```

Comment:
```suggestion
                    (availableAction.type === 'opsgenie' ||
                      availableAction.type === 'pagerduty') ? (
```
also suggest moving these `opsgenie` and `pagerduty` to constants at the top of the file:
```
const OPSGENIE = 'opsgenie';
const PAGERDUTY = 'pagerduty';
```